src/test/java/io/supertokens/test/session/api/SessionGetAPIJWTTest.java

```diff
@@ -0,0 +1,164 @@
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+package io.supertokens.test.session.api;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import io.supertokens.ProcessState;
+import io.supertokens.session.Session;
+import io.supertokens.test.TestingProcessManager;
+import io.supertokens.test.Utils;
+import io.supertokens.test.httpRequest.HttpRequest;
+import io.supertokens.test.httpRequest.HttpResponseException;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertNotNull;
+
+public class SessionGetAPIJWTTest {
+    @Rule
+    public TestRule watchman = Utils.getOnFailure();
+
+    @AfterClass
+    public static void afterTesting() {
+        Utils.afterTesting();
+    }
+
+    @Before
+    public void beforeEach() {
+        Utils.reset();
+    }
+
+    @Test
+    public void testAPIFailuresWithBadInputParameters() throws Exception {
+        String[] args = {"../"};
+
+        TestingProcessManager.TestingProcess process = TestingProcessManager.start(args);
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STARTED));
+
+        //Get Request input errors
+        //null is sent in parameters
+        try {
+            io.supertokens.test.httpRequest.HttpRequest
+                    .sendGETRequest(process.getProcess(), "", "http://localhost:3567/recipe/session", null, 1000,
+                            1000,
+                            null, Utils.getCdiVersionLatestForTests(), Session.RECIPE_ID);
+            fail();
+        } catch (HttpResponseException e) {
+            assertTrue(e.statusCode == 400 && e.getMessage()
+                    .equals("Http error. Status Code: 400. Message: Field name 'sessionHandle' is missing in GET " +
+                            "request"));
+        }
+
+        //typo in parameter
+        HashMap<String, String> map = new HashMap<>();
+        map.put("sessiondle", "");
+
+        try {
+            io.supertokens.test.httpRequest.HttpRequest
+                    .sendGETRequest(process.getProcess(), "", "http://localhost:3567/recipe/session", map, 1000,
+                            1000,
+                            null, Utils.getCdiVersionLatestForTests(), Session.RECIPE_ID);
```

Comment:
Use `"session"` instead of `Session.RECIPE_ID`

Reply:
Done